utils/global-exception-handling.js

```diff
@@ -0,0 +1,18 @@
+const unhandledRejection = process.on('unhandledRejection', err => {
+	console.error((new Date).toUTCString() + ' unhandledRejection:', err.message)
+	console.error(err.stack)
+});
+
+const unhandledException = process.on('unhandledException', err => {
+	console.error((new Date).toUTCString() + ' unhandledException:', err.message)
+	console.error(err.stack)
+});
+
+const globalExceptionHandling = function () {
+	unhandledException;
+	unhandledRejection
+};
+
+module.exports = {
+	globalExceptionHandling
```

Comment:
Make and export two functions - `handleRejection(callback)` and `handleException(callback)` where the caller of these method passes the callback functions. In these two functions add the process.on listener with the provided callback. Require them in the cli.js, pass the callback functions and you are done